src/components/state/models/staticCandidateCards.js

```diff
@@ -0,0 +1,9 @@
+export const staticCandidateCards = [
```

Comment:
we can import candidate image to here and pass through here right?

Reply:
you right) I forgot that I have created this file already